src/binaries/BinariesManager.ts

```diff
@@ -8,7 +8,6 @@ import { Constants } from '../Constants';
 import { Logger } from '../logger/Logger';
 import { TelemetryEvent } from '../logger/TelemetryEvent';
 import { AccountContextManager } from '../models/context/AccountContextManager';
-import { BinariesUtility } from './BinariesUtility';
 import { BinariesUtilityV2 } from './BinariesUtilityV2';
 import { IBinariesUtility } from './IBinariesUtility';
```

Comment:
Are there any reasons to keep the other BinariesUtility class?

Reply:
To keep things simple I have not deleted this class but we will be delating this classes and also rename all the classes with V2 to indicate only one version